crates/ruff_linter/src/rules/pyflakes/rules/unused_import.rs

```diff
@@ -329,6 +336,7 @@ pub(crate) fn unused_import(checker: &Checker, scope: &Scope, diagnostics: &mut
             import,
             range: binding.range(),
             parent_range: binding.parent_range(checker.semantic()),
+            in_global_scope,
```

Comment:
I'm inclined to store the `ScopeId` instead. It seems more generally useful and is cheap as well

Reply:
And it is more strongly typed/more readable than passing around a boolean value